src/app/create-event/test/page.tsx

```diff
@@ -0,0 +1,17 @@
+export default function Test() {
```

Comment:
Is this file needed?

Reply:
Removed the file I was using for testing.